docs/stories/04-components/TextButton.mdx

```diff
@@ -0,0 +1,81 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import { TextButton } from '@strapi/design-system';
+
+import * as TextButtonStories from './TextButton.stories';
+
+<Meta of={TextButtonStories} />
+
+# TextButton
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Props](#props)
+- [Variants](#variants)
+- [Usage with other routing libraries](#usage-with-other-routing-libraries)
+
+## Overview
+
+Typically used for internal navigation within a Strapi application. These buttons are not supposed to have "actions"
+e.g. clone a document.
+
+<ViewSource path="components/TextButton" />
+
+## Usage
+
+```ts
+import { TextButton } from '@strapi/design-system';
+```
+
+<Canvas of={TextButtonStories.Base} />
+
+## Props
+
+<TypeTable of={TextButton} />
+
+## Variants
+
+### Disabled
+
+<Canvas of={TextButtonStories.Disabled} />
+
+### Loading
+
+<Canvas of={TextButtonStories.Loading} />
+
+### With start icon
+
+<Canvas of={TextButtonStories.StartIcon} />
+
+### With end icon
+
+<Canvas of={TextButtonStories.EndIcon} />
+
+## Usage with other routing libraries
+
+### React Router
+
+To use the LinkButton component with a routing library (e.g. react-router-dom), you'll need to pass the `NavLink`
+component to the `tag` prop in order to replace the default HTML anchor `<a />`. You'll now be able to pass all
+`NavLink` props.
```

Comment:
Does this mean NavLink will by default render an anchor if you don't pass a tag? I'm confused as nothing about TextButton implies link behavior to me. I would expect the HTML to be just a button

Reply:
Yeah I don't like the name either, but i'm just matching Product naming. The idea for them supposedly is not to be used as buttons but only for navigation 🤪 we can raise this with product and deprecate during v2 lifecycle if they agree?

Reply:
that sounds good!